crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,6 +1,26 @@
+import glob
+import random
+
+
+def read_names():
+    """
+    Read the names in ../names
+    :return: names_list
+    """
+    names_list = []
+    for name_file in glob.glob("./names/*"):
+        with open(name_file, 'r') as f:
+            first_line = f.readline()
+            first_name = first_line.split()[0]
+            names_list.append(first_name)
+    return names_list
+
+
+def greet(names):
+    greeting_list = ["Hello", "Hi", "Howdy","Hola","Ciao"]
+    print("%s, %s!" % (random.choice(greeting_list),
```

Comment:
I strongly recommend using f-strings to make your code more readable.  This would look like
```
print(f"{greeting_choice} {name_choice})
```